### DISCUSSION TITLE
Regaining access to a Github user account which shared an email address with another account

### DISCUSSION THREAD

**Discussion by vonmises** [community/community#22308] [New to GitHub]
<p>Hi,</p>
<p>It probably shouldn’t have been possible–and I’m not sure whether it still is–but I was able to create two Github user accounts years ago using the same email address.</p>
<p>It was never a problem since I usually don’t forget my password, but this time I did, after not working for the last month or so. I used the password reset feature because I just needed to get up and running (I didn’t think of the multiple-account/single email address issue since I never even used the other account).</p>
<p>When Github sent me the password reset token/link, it redirected into my other account and now I can’t get access into my primary account (which I finally found the password to).</p>
<p>When I try to log in, It doesn’t tell me the account is locked or suspended (which it did before I did the password reset, on account of multiple unsuccessful tries). It just says incorrect username or password.</p>
<p>Is there a way to get access into the other account? I’ve already contacted Github support about 8 hours ago but I don’t know how long it’ll take them to respond or even acknowledge the email.</p>
<p>I’ve searched online and in this forum and I don’t see issues which even approach this one.</p>
<p>Thanks.</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/vonmises">@vonmises</a></p>
<p>It really is not possible to create two accounts with the exact same mail address. Possibly you have used some sort of alias for the other account, or an <a href="mailto:x+y@domain.com">x+y@domain.com</a> address.</p>
<p>Anyway, there’s not much we can help with on the forum, only support can help. They try to answer all questions within 24 hours.</p>

**Comment by vonmises** [comment]
<blockquote>
<p><a class="mention" href="/seveas">@seveas</a> wrote:</p>
</blockquote>
<blockquote>
<p>Hi <a class="mention" href="/vonmises">@vonmises</a></p>
</blockquote>
<blockquote>
<p>It really is not possible to create two accounts with the exact same mail address. Possibly you have used some sort of alias for the other account, or an <a href="mailto:x+y@domain.com">x+y@domain.com</a> address.</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>Anyway, there’s not much we can help with on the forum, only support can help. They try to answer all questions within 24 hours.</p>
</blockquote>
<p>this was the issue:</p>
<p>“Possibly you have used some sort of alias for the other account, or an <a href="mailto:x+y@domain.com">x+y@domain.com</a> address.”</p>
<p>thanks for the quick response, as well. not only support can help.</p>

**Comment by apple3feet** [comment]
I have two accounts with the same email address - I created them years ago. I have verified by looking through old emails that they are using the same email address.

Now I can't log in to one because its password is not complex. But I can't change its password because the reset email changes the password of the other account. I have submitted a support ticket.

Has anyone had this experience?

**Reply by d4yn1sh** [reply]
Were you able to resolve this? I have an old account I want to access but can't for the same reason
